### PR TITLE
Add NetworkKey file and openssl ref.

### PR DESCRIPTION
**Pull request by YJSoft**
Also changing library sha256 since newer library zip is needed.

### CODE REVIEW

**Comment by janisozaur** [comment]
Thank you.

I noticed, however, you have not changed https://github.com/OpenRCT2/OpenRCT2/blob/e7510262aca54e8ec549d91951cad17b11126514/OpenRCT2.xcodeproj/project.pbxproj#L306-L323. As far as I know the builds on OS X require embedding shared objects into resulting zip so it can be properly redistributed. I don't know structure of the project file and it is only my educated guess, this bit should get modified as well, but can you confirm which way is it?

File: OpenRCT2.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2155,6 +2162,8 @@
 					rct2_data,
 					rw,
 					rw,
+					"-lssl",
```
Comment (janisozaur) [inline OpenRCT2.xcodeproj/project.pbxproj]: All the other libs used for linking are listed elsewhere, please follow their convention

**Comment by YJSoft** [comment]
@janisozaur Done

**Comment by janisozaur** [comment]
Great, thanks.

I noticed builds done with your updated libs are failing, my theory is following:
you use `-sectcreate` on OS X, which creates a section at specified VMA, but you never move `.text` and others, they still load at `0x1000` and they have eventually depleted the space available until `0x401000`, which is where rct2 sections start.

can you double-check if everything looks good on your end with those libs?

I invite you to join us on gitter, where we usually hang out, but I'm in CEST zone, so I'm almost done for the day.

**Pull request merged**